domains/mayank.json

```
@@ -4,6 +4,11 @@
     "email": "mayankreal657@gmail.com"
   },
   "records": {
-    "CNAME": "mayankservers.github.io"
+    "CNAME": "mayankservers.github.io",
+    "MX": [
+      { "target": "mx1.improvmx.com", "priority": 10 },
+      { "target": "mx2.improvmx.com", "priority": 20 }
+    ],
+    "TXT": "v=spf1 include:spf.improvmx.com ~all"
   }
```

```suggestion
"records": {
    "CNAME": "mayankservers.github.io",
    "MX": [
      { "target": "mx1.improvmx.com", "priority": 10 },
      { "target": "mx2.improvmx.com", "priority": 20 }
    ],
    "TXT": "v=spf1 include:spf.improvmx.com ~all"
},
"proxied": true
```
You can't combine CNAME with other records unless it was proxied

---

Solved